packages/services/f-analytics/src/mixins/analytics.mixin.vue

```diff
@@ -62,13 +66,18 @@ export default {
             platformData.language = COUNTRY_INFO[this.$i18n.locale].language;
             platformData.currency = COUNTRY_INFO[this.$i18n.locale].currency;
 
-            this.updatePlatformData(platformData);
+            this.pushPlatformData(platformData);
```

Comment:
Do we still need this preparePlatformData() method if you moved it to service in this PR? Sorry, maybe I missed something

Reply:
Left it there to reduce merge issues because @AndyMayJE relies on the method atm, so if I remove it in my pr then I cannot see what logic I need to re-add to the service method.

Reply:
Just until @AndyMayJE merges his pr because he has logic to add to this area and if I delete it on my pr then merge hell :\